profile/README.md

```diff
@@ -0,0 +1,61 @@
+## 👋 Welcome to the [Adyen](https://www.adyen.com/) GitHub organization 👋
+
+<!-- ![The Adyen Logo](https://github.com/adyen-examples/.github/raw/main/images/logo.png) -->
+
+<img src="https://github.com/adyen-examples/.github/raw/main/images/logo.png" height="100" alt="The Adyen logo">
+
+This page contains the source code for all of our libraries and plug-ins as well as our Open API specifications. To find a list of example integrations for those, you can visit **[the Adyen examples GitHub page](https://github.com/adyen-examples)**.
+
+## 📜 Our documentation pages : 
+
+* [Adyen Technical documentation](https://docs.adyen.com/)
+* [Adyen API Explorer](https://docs.adyen.com/api-explorer/)
+* [Adyen Test Cards](https://docs.adyen.com/development-resources/test-cards/test-card-numbers/)
+
+## 👩‍💻 Useful resources: 
+
+* 📝 [Our technical blog](https://dev.to/adyen)
+* 🐦 [Our Developer Twitter account](https://twitter.com/AdyenDevs)
+* ❓ [Ask a question on Stack Overflow](https://stackoverflow.com/questions/tagged/adyen)
+* 📰 [Our developer newsletter](https://www.adyen.com/newsletter/developers)
+* 📺 [Checkout our Tech playlist on Youtube](https://www.youtube.com/watch?v=VPpTgsJbIhc&list=PL6agz7H5yEoaS-bF2gIwRwe_ApzqmW_QX)
+
+**🌈 Contributions are most welcome so feel free to reach out, post issues and / or propose improvements🦄**
+
+## Currently available resources
+
+### Libraries and SDKs
+
+* [Adyen Web Drop-in and Components](https://github.com/Adyen/adyen-web)
```

Comment:
somehow the markup of this list is different then the one below

Reply:
Hum, I don't see what you mean. Can you propose a change maybe?